lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextLifeCycle.cs

```diff
@@ -71,10 +71,10 @@ public void Start()
             IContextStart contextStart = new ContextStartImpl(Id);
             
             // TODO: enable
-            //foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
-            //{
-            //    startHandler.OnNext(contextStart);
-            //}
+            //// foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
```

Comment:
If you try three lines until the indentation of `startHandler`, you will get errors.
```
// foreach
// {
//     startHandler....
```
StyleCop enforce a sinle space after '//'. The guideline of StyleCop to write `comment-outed code` says the use of '////'. Please refer the following guideline.
http://stylecop.soyuz5.com/SA1005.html